src/components/Modal/Modal.js

```diff
@@ -23,6 +23,7 @@ export default class Modal extends Component {
     primaryButtonDisabled: PropTypes.bool,
     onSecondarySubmit: PropTypes.func,
     danger: PropTypes.bool,
+    enterSubmit: PropTypes.bool,
```

Comment:
How about `shouldSubmitOnEnter` to follow the pattern in some other components? 😄